src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap';
+
+$main-accent: #060B35;
+$secondary: #616070;
+$blue-accent: #00ACDC;
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin: 0 auto;
+
+  width: 200px;
+  height: 408px;
+
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__imac {
```

Comment:
The image has an additional 3px indent in the line
![image](https://github.com/mate-academy/layout_product-cards/assets/64101898/0691f03c-a501-4d66-a622-cd487bd69e90)